.github/workflows/ci.yml

```diff
@@ -18,9 +18,10 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
     steps:
       - uses: actions/checkout@v4.1.1
-      - name: Set up Python 3.8
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Unrelated "bugfix"

Reply:
This will make the tox lint job also run on all Python versions. We normally only run the lint job on the minimum Python version to avoid conflicts in the linting behavior between different Python versions. For tox there's the tox-gh-actions tool that can be used for matrix jobs.

Example:
https://github.com/home-assistant-libs/pytradfri/blob/63d1524eea56fa79467e72ea7355c6a369df96f2/tox.ini#L5-L8

https://github.com/home-assistant-libs/pytradfri/blob/63d1524eea56fa79467e72ea7355c6a369df96f2/.github/workflows/test.yml#L11-L31

Reply:
It already does that.
This change only changes the name of this job to reflect what it actually is doing.

Reply:
Right. :+1:

Reply:
Adjusted here <https://github.com/NabuCasa/snitun/pull/220>